scripts/hub_token.sh

```
@@ -0,0 +1,9 @@
+#!/bin/bash
+export HUB_TOKEN=$(curl -sSf -k --resolve "tackle.local:8443:127.0.0.1" \
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)